src/value-converters/sanitize-html-html-sanitizer.ts

```diff
@@ -0,0 +1,46 @@
+import sanitize from 'sanitize-html'
+import {HTMLSanitizer} from "aurelia-templating-resources"
+
+/**
+ *  Sanitizes HTML tags for use in `innerhtml` attribute.
+ *  <p>You have to configure Aurelia DI to use an implementation on configuration stage.</p>
+ *  <p>Usage: <code><span innerhtml.bind="insecureValue|sanitizeHTML"></span></code></p>
+ * @author Simon Beyer <Simon.Beyer@t-systems.com>
+ */
+export class SanitizeHtmlHtmlSanitizer implements HTMLSanitizer {
+    private _options
+
+    constructor() {
+        this.restoreDefaults()
+    }
+
+    get options(): sanitize.IOptions {
```

Comment:
Its not a good practice in the "Tell, don't ask priciple" (https://clean-code-developer.de/die-grade/gruener-grad/#Tell_don8217t_ask) to read the options from the class. It would be better to just write these things like:

```typescript
const sanitizer = new SanitizeHtmlHtmlSanitizer();
sanitizer.setOptions({
    disallowedTagsMode: "recursiveEscape",
});

Reply:
Mmh, unconvinced. I'd say Setter is the-same-but-worse-than Getter, as far as object orientation is concerned.
At first I wanted to provide methods for all the options, but there are a lot of them (and of course, our middleware code does not actually care about them). By providing the object, user code automatically has all the options _and current values_ their installed peer dependency supports/provides.

Encountering this setter pattern in user code, I _always_ end up having to debug into the library code to look what happens. That may be information hiding alright, but it is not _useful, helpful or nice_ .

... But can do.

Reply:
The user then _must_ replace all options. Or I _must_ add brittle logic to treat missing fields as "keep previous".